policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/base.yaml

```diff
@@ -0,0 +1,11 @@
+description: Manitoba provides this base amount for its Child Tax Benefit.
+values:
+  2023-01-01: 420 #35 per month
```

Comment:
need to add 2021 and 2022 values? If Manitoba does not have that value in 2021 and 2022, 2021-01-01: 0